src/sc2laddercore/Proxy.cpp

```diff
@@ -460,9 +450,9 @@ void Proxy::gameUpdate()
     PrintThread{} << m_botConfig.BotName << " : Exiting with " << GetExitCaseString(m_result) << " Average step time " << m_stats.avgLoopDuration << " microseconds, total time: " << std::chrono::duration_cast<std::chrono::seconds>(m_totalTime).count() << " seconds, game loops: " << m_currentGameLoop << std::endl;
 }
 
-bool Proxy::isBotCrashed(const int millisecons) const
+bool Proxy::isBotCrashed(const int milliseconds) const
```

Comment:
There is no real need in `const int` because `int` is always passed as a copy.

Reply:
Interesting that you say this here. I mean in the cpp it is more "this value won't change in the function body" semantic info. So a thing of personal taste.
In the header, however, I recently learned that there is actually the recommendation to not use `const` on copies because then you can change it whenever you like in the cpp without needing to touch the header.
Anyway, I spent all these years to get into the habit of putting `const` everywhere and now this :(